src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import { taskStatusToCopy } from "constants/task";
+import { TaskStatus } from "types/task";
+
+interface Props {
+  status: TaskStatus;
+  count: number;
+}
+export const GroupedTaskStatusBadge: React.FC<Props> = ({ count, status }) => {
+  let statusDisplayName = taskStatusToCopy[status];
+  if (statusDisplayName.slice(-1) === "s" && count !== 1) {
+    statusDisplayName += "es";
+  } else if (statusDisplayName.slice(-1) === "e" && count !== 1) {
+    statusDisplayName += "s";
+  }
+
+  return (
+    <BadgeContainer css={statusToColorVariant[status]}>
```

Comment:
This component should also be a link. Since it should link to the version page with the filters applied. 
I think it would make sense for this to accept two optional props.

`onClick` and `to` 
`to` enabling this to become a custom react-router <Link /> element.

Reply:
Sounds good!